src/file.rs

```diff
@@ -428,11 +439,7 @@ fn blend_mode_to_blend_fn(mode: BlendMode) -> BlendFn {
     }
 }
 
-fn tile_pixels<'a>(
-    pixels: &'a [pixel::Rgba],
-    tile_size: &TileSize,
-    tile_id: &TileId,
-) -> &'a [pixel::Rgba] {
+fn tile_slice<'a, T>(pixels: &'a [T], tile_size: &TileSize, tile_id: &TileId) -> &'a [T] {
```

Comment:
A comment explaining the intent of this function would be nice. It's not that obvious from the name & type.